.github/workflows/benchmarks.yml

```diff
@@ -7,125 +7,85 @@ permissions:
   contents: read
 
 jobs:
-  benchmarks-ubuntu:
-    name: Ubuntu
+  benchmarks:
+    name: Benchmarks
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      HWLOC_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows"
     strategy:
       matrix:
-        os: ['ubuntu-22.04']
-        build_type: [Release]
-        compiler: [{c: gcc, cxx: g++}]
-        shared_library: ['ON', 'OFF']
+        include:
+          - os: ubuntu-latest # shared library on Linux
+            shared_library: 'ON'
+            pool_scalable: 'ON'
+            pool_jemalloc: 'ON'
+
+          - os: ubuntu-latest # static library on Linux
+            shared_library: 'OFF'
+            pool_scalable: 'ON'
+            pool_jemalloc: 'ON'
+
+          - os: windows-latest # shared library on Windows
+            shared_library: 'ON'
+            pool_scalable: 'OFF'
+            pool_jemalloc: 'OFF'
+
+          - os: windows-latest # static library on Windows
+            shared_library: 'OFF'
+            pool_scalable: 'OFF'
+
     runs-on: ${{matrix.os}}
 
     steps:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Install apt packages
+        if: matrix.os == 'ubuntu-latest'
         run: |
           sudo apt-get update
           sudo apt-get install -y cmake libhwloc-dev libnuma-dev libjemalloc-dev libtbb-dev
 
+      - name: Initialize vcpkg
+        if: matrix.os == 'windows-latest'
+        uses: lukka/run-vcpkg@5e0cab206a5ea620130caf672fce3e4a6b5666a1 # v11.5
+        with:
+          vcpkgGitCommitId: 3dd44b931481d7a8e9ba412621fa810232b66289
+          vcpkgDirectory: ${{env.BUILD_DIR}}/vcpkg
+          vcpkgJsonGlob: '**/vcpkg.json'
+
+      - name: Install dependencies
```

Comment:
you can rename it to e.g. `Install vcpkg packages` (to be aligned in name with the `apt` step)

Reply:
Done